core/src/datayoga_core/expression.py

```diff
@@ -14,6 +15,7 @@
 from typing import Any, Dict, List, Tuple, Union
 
 import jmespath
+
```

Comment:
not need, use the same extension we use in the project.

Reply:
what isn't used?

Reply:
empty line, I removed it and pushed.